# Tag message structure. qf is the quality of this measurement.

Header header

float64 x ## in m
float64 y ## in m
float64 z ## in m
float64 qf ## in percent
int16 n_anchors ## number of anchors

